Java/core/src/main/java/com/amazon/randomcutforest/imputation/ImputeVisitor.java

```diff
@@ -179,6 +215,14 @@ public boolean trigger(final INodeView node) {
         return missing[node.getCutDimension()];
     }
 
+    public double getRank() {
```

Comment:
Given that we changed `rank` to `anomalyRank` for clarity, should we change the getter to match?

Reply:
Yes, and perhaps make the modifier protected since the main goal foreseen is in eventual subclassing.